cmd/create.go

```diff
@@ -210,6 +239,12 @@ func (cc *createCmd) generateDockerfile(langConfig *config.DraftConfig, lowerLan
 		}
 	}
 
+	if cc.dryRunWriter != nil {
```

Comment:
can we get rid of the cc.dryRunWriter field somehow? feel like it doesn't make sense to store the same object in multiple fields (templateWriter and dryRunWriter). Maybe just use a flag/boolean like isDryRun instead? If not no worries

Reply:
After our discussion I am quite stumped on how we would do this but will keep thinking about it lol